firmware/controllers/sensors/AemXSeriesLambda.cpp

```diff
@@ -12,12 +12,43 @@ AemXSeriesWideband::AemXSeriesWideband(uint8_t sensorIndex, SensorType type)
 	, m_sensorIndex(sensorIndex)
 {}
 
+bool AemXSeriesWideband::acceptFrame(const CANRxFrame& frame) const {
+	uint32_t id = CAN_ID(frame);
+
+	// 0th sensor is 0x180, 1st sensor is 0x181, etc
+	uint32_t aemXSeriesId = 0x180 + m_sensorIndex;
+
+	// 0th sensor is 0x190 and 0x191, 1st sensor is 0x192 and 0x193
+	uint32_t rusefiBaseId = 0x190 + 2 * m_sensorIndex;
+
+	return 
+		id == aemXSeriesId ||
+		id == rusefiBaseId ||
+		id == rusefiBaseId + 1;
+}
+
 void AemXSeriesWideband::decodeFrame(const CANRxFrame& frame, efitick_t nowNt) {
 	if (frame.DLC != 8) {
 		invalidate();
 		return;
 	}
 
+	int32_t id = CAN_ID(frame);
+
+	// accept frame has already checked if the message belongs to
+	// this sensor index to us, we just have to check if it's AEM or rusEFI
+	if (id <= 0x190) {
```

Comment:
https://blog.devgenius.io/code-smell-02-constants-and-magic-numbers-d6c320eef90b